CHANGELOG.md

```diff
@@ -112,40 +125,50 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * The `diff.stat()` template method now provides methods to get summary values.
 
-* New `git.sign-on-push` config option to automatically sign commits which are being
-  pushed to a Git remote.
+* Add templater support for rendering commit signatures.
+
+* Add a config option `ui.show-cryptographic-signatures`. When set to `true`,
+  the builtin templates will show signataure information if available. The
+  signature display can be customized using
+  `format_detailed_cryptographic_signature(signature)` and
+  `format_short_cryptographic_signature(signature)`.
+
+* New `git.sign-on-push` config option to automatically sign commits which are
+  being pushed to a Git remote.
 
 * New `git.push-new-bookmarks` config option to push new bookmarks without
   `--allow-new`.
 
+* `jj status` now shows untracked files when they reside directly under a
+  tracked directory.
```

Comment:
It's probably helpful link to #5389 so the reader doesn't wonder if the described behavior is intentional

Reply:
Added `There's still an issue that files under untracked
  directories aren't listed. [#5389](https://github.com/jj-vcs/jj/issues/5389)`